src/client/classes/AppMessage.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * Possible types of message to display to the user
+ */
+
+export enum MessageType {
```

Comment:
Can I suggest making these block capitals (like constants)?

i,e:

```suggestion
export enum MESSAGE_TYPE {
  ERROR = 'error',
  SUCCESS = 'success',
  INFO = 'info',
}
```

Reply:
Makes sense. updated in 3394981